flixel/system/replay/FrameRecord.hx

```diff
@@ -60,6 +60,9 @@ class FrameRecord
 	public function save():String
 	{
 		var output:String = frame + "k";
+	
+		if ((keys == null) && (mouse == null))
```

Comment:
It was apparently originally (before this PR) written with this purpose in mind, to [wait during frames not mentioned in the save file](https://github.com/HaxeFlixel/flixel/blob/dev/flixel/system/replay/FlxReplay.hx#L198) as I mentioned here about [the old loader being able to load the new format](https://github.com/HaxeFlixel/flixel/pull/1733#issuecomment-186262691).